### DISCUSSION TITLE
Issues finding my other pc in NDI sources

### DISCUSSION THREAD

**Discussion by TheGarbi** [DistroAV/DistroAV#818] [Q&A - Support] [answered]
The pc that is supposed to be my gaming pc I am able to find both computers in the sources. My "streaming" pc is not able to find the other computer in the sources. 
Even when I try to use the streaming pc to send some footage to the gaming pc to test no image shows up. 
As I am not the greatest computer person I have already looked for solutions to my problem but I am not able to exactly pinpoint my issue nor find an adequate solution to my issue. 
I am using the 4.10.0 version of obs-NDI (used the installer and downloaded NDI runtime) and the version 28.1.2 of obs.
If anyone has any idea on how to fix my issues <3 happy holidays!

**Comment by Trouffman** [comment, marked as answer]
To debug, make sure that first the 2 PC sees each-other usign the NDI Tools : 


First, update the NDI Runtime to the latest at : http://new.tk/NDIRedistV5

Then
Do the following to test:

- [ ] make a new scene collection
- [ ] only one scene
- [ ] only one source (your NDI phone source)
- [ ] Set the source to be the full size (transfrom > fit to screen/canva)
- [ ] nothing else

Test if working.

If not, find out if this is the plugin or something else NDI related.

- [ ] Install the NDI Tools from - https://downloads.ndi.tv/Tools/NDI%205%20Tools.exe
- [ ] Open NDI Studio Monitor & Test Pattern (on both PC)
- [ ] Check if the other PC sees the test pattern fromt he other PC

If working, this might be somethign to heklp with (plugin /configuration related)

If not working : this is soemthing not fromt he plugin but NDI is not commmunicating in between.
